Paintroid/src/main/java/org/catrobat/paintroid/ui/viewholder/BottomNavigationViewHolder.java

```diff
@@ -25,22 +25,28 @@
 import android.support.design.widget.BottomNavigationView;
 import android.view.LayoutInflater;
 import android.view.Menu;
+import android.view.MenuItem;
 import android.view.View;
 import android.widget.ImageView;
 import android.widget.TextView;
 
 import org.catrobat.paintroid.R;
 import org.catrobat.paintroid.contract.MainActivityContracts;
+import org.catrobat.paintroid.tools.ToolType;
+
+import static android.content.pm.ActivityInfo.SCREEN_ORIENTATION_PORTRAIT;
 
 public class BottomNavigationViewHolder implements MainActivityContracts.BottomNavigationViewHolder {
```

Comment:
Can you extract a class for each orientation instead of switching over it several times? (strategy pattern)